src/main/scala/org/phenoscape/kb/ingest/bgee/BgeeExpressionToOWL.scala

```diff
@@ -0,0 +1,78 @@
+package org.phenoscape.kb.ingest.bgee
+
+import scala.collection.JavaConversions._
+import scala.collection.mutable
+import java.io._
+import scala.io.Source
+import org.phenoscape.scowl.Functional._
+
+import org.apache.commons.lang3.StringUtils
+import org.phenoscape.kb.ingest.util.Vocab._
+import org.phenoscape.kb.ingest.util.ExpressionUtil
+import org.phenoscape.kb.ingest.util.OBOUtil
+import org.phenoscape.kb.ingest.util.OntUtil
+import org.phenoscape.scowl._
+import org.semanticweb.owlapi.model.OWLAxiom
+import org.semanticweb.owlapi.model.IRI
+
+object BgeeExpressionToOWL {
+
+  def strToSource(str: String): Source = {
+    io.Source.fromFile(str)
+  }
+
+  def convert(expressionData: Source): Set[OWLAxiom] = expressionData.getLines.flatMap(translate).toSet[OWLAxiom]
+
+  def translate(expressionLine: String): Set[OWLAxiom] = {
+    val items = expressionLine.split("\t", -1)
+    
+    if (items.length == 1 || items(6).startsWith("absent") || items(6).startsWith("Expression") || items(2).startsWith("ZFA")) {
```

Comment:
@calliem could you add a comment here that we are skipping ZFA terms because these are all embryonic or otherwise pre-adult developmental stages that wouldn't make sense with the data the KB is otherwise populated with? Otherwise this reason may be lost when someone looks at it in the future.